fpsim/locations/senegal/optimize-space-prefs.py

```diff
@@ -7,49 +7,52 @@
 import pylab as pl
 import fpsim as fp
 
-# Define upper and lower limits for the parameters
+# Define best, low, high limits for the parameters
 calib_pars = dict(
-    space0_6=[0.5, 0.2, 1.5],
-    space9_15=[0.5, 0.2, 1.5],
-    space18_24=[2.0, 0.5, 3.0],
-    space27_36=[1.0, 0.5, 3.0]
+    space0_6   = [0.5, 0.2, 1.5],
+    space9_15  = [0.5, 0.2, 1.5],
+    space18_24 = [2.0, 0.5, 3.0],
+    space27_36 = [1.0, 0.5, 3.0],
 )
 
 spacing_file = 'BirthSpacing.obj'
 
-spacing_bins = sc.odict({'0-12': 0, '12-24': 1, '24-48': 2, '>48': 4})  # Spacing bins in years
+spacing_bins = sc.odict({'0-12': 0, '12-24': 1, '24-48': 2, '>48': 4})  # Spacing bins in years; CK: need to check that this is right
```

Comment:
This was actually intentional.  The keys are labeling the number of years each space falls within for categorization.  So anything 0-1 should fall in bin 0-12 months, 1-2 should be 12-24 months, 2-4 is 24-48 months, and over 4 is > 48 months.  

This is assuming that this code is in fact categorizing that way: 
            for d in range(len(ppl.dobs[i]) - 1):
                space = ppl.dobs[i][d + 1] - ppl.dobs[i][d]
                ind = sc.findinds(space > spacing_bins[:])[-1]
                model_spacing_counts[ind] += 1

I admit to having to think my into that second line inside the for loop many times to convince myself it's categorizing correctly.